website/content/docs/v0.14/Cloud Platforms/oracle.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Oracle"
```

Comment:
this should go under v0.15 once we cut them (I will try to do this today)

Reply:
Yep, definitely